test_pull_requests

```diff
@@ -550,9 +698,9 @@ module Hub
       update_status(settings['test_prefix'], commit['sha'], repo, 'success', build_url, 'Passed')
       sleep 10
```

Comment:
Little weird that this will be the only unparameterized `sleep` after this PR -- do we have a good reason not to use `SLEEP_TIME` here with maybe a different number of retries? Does the status update task take longer to be reflected in the API?